ubtue_Philosophy Documentation Center.js

```diff
@@ -0,0 +1,63 @@
+{
+	"translatorID": "5e3f67c9-f4e5-4dc6-ad9a-93bf263a585a",
+	"label": "Philosophy Documentation Center",
+	"creator": "Madeesh Kannan",
+	"target": "^https://www.pdcnet.org/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": false,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2018-11-02 15:40:06"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2018 Universitätsbibliothek Tübingen.  All rights reserved.
+
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+function detectWeb(doc, url) {
```

Comment:
Many subpages do not contain bibliographic information and should not be used with zotero, e.g. [Memberships](https://www.pdcnet.org/wp/memberships/). Please consider adding a (positive) URL filter.